### PR TITLE
Add documentation for user admin server

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -384,6 +384,18 @@ Same as study notification server, but for GridMerge asynchronous update.
 
 This is the only entry point to the back-end. Front-ends can only send requests to the gateway. All the other web services do not expose their API directly to the front-ends. Requests are routed by the gateway to other micro services. This gateway is mainly responsible from implementing secury features: https and user access right verification.
 
+### User admin server
+
+- Kind: Web service with a REST API
+- Source repository: https://github.com/gridsuite/user-admin-server
+- Storage: PostgreSQL
+- Other services dependencies: no
```
Comment (sBouzols) [inline README.md:392]: - Connected to message broker: no

File: README.md [hunk 2]
```diff
@@ -384,6 +384,18 @@ Same as study notification server, but for GridMerge asynchronous update.
...
+- Use PowSyBl libraries: no
+
+This service is responsible for storing a list of users authorized to access the application and storing also users connection information.
+It is used by the gateway service to check access authorization.
```
Comment (sBouzols) [inline README.md:396]: Then should we add it in Gateway.Other services dependencies ?
Not sure...

Comment (sBouzols) [reply README.md:396]: Not sure because actually Gateway has everything as dependency then...

File: README.md (hunk 1 above)
Comment (FranckLecuyer) [reply README.md:393]: Added

File: README.md (hunk 2 above)
Comment (FranckLecuyer) [reply README.md:397]: changed

**Review by sBouzols — APPROVED** [review]

**Review by bendaamerahm — APPROVED** [review]

**Pull request merged**